packages/react/src/components/Storage/FileUploader/hooks/useFileUploader/types.ts

```diff
@@ -0,0 +1,18 @@
+import { Files } from '../../types';
+
+export interface DragActionHandlers {
+  onDragStart: (e: React.DragEvent<HTMLDivElement>) => void;
+  onDragEnter: (e: React.DragEvent<HTMLDivElement>) => void;
+  onDragLeave: (e: React.DragEvent<HTMLDivElement>) => void;
+  onDrop: (e: React.DragEvent<HTMLDivElement>) => void;
+  onDragOver: (e: React.DragEvent<HTMLDivElement>) => void;
```

Comment:
NIT, and this can be on the other PR you created, but we can rename `e` -> `event` for explicitness

Reply:
This will be added in the next PR